Sources/Fluent/Memory/Memory+Filters.swift

```diff
@@ -148,6 +148,10 @@ extension Node {
             case .or:
                 return passesOne(filters)
             }
+        case .raw(command: let command, values: let values):
+            // TODO: Throw error, memory DB doesn't support raw?
```

Comment:
The reason I thought maybe it shouldn't throw an error is if users want to use raw in some way w/ memory db just for testing, but I guess this wouldn't work either way, unless we want to build a SQL interpreter ... which I do not, so throwing is probably the expectation